### PR TITLE
Align query for nightly_regressions_unassigned_p_high

### PR DESCRIPTION
**Pull request by apiraino**
Hi,

I think the query for the `nightly_regressions_unassigned_p_high` issues in the template is not aligned to the agenda template. The github query filters out all teams, the template doesn't so the results are not aligned if I click on the link or run the agenda generator using the cli tool.

I think the Github query is the right one so I fixed the agenda template: we want all `P-high` regressions on nightly that are not yet assigned to a team, right?

r? @Mark-Simulacrum 
cc: @spastorino

### CODE REVIEW

File: templates/prioritization_agenda.tt [hunk 1]
```diff
@@ -93,7 +93,7 @@ tags: weekly, rustc
 [P-high beta regressions](https://github.com/rust-lang/rust/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3Aregression-from-stable-to-beta+label%3AP-high)
 {{-issues::render(issues=beta_regressions_p_high, empty="No `P-high` beta regressions this time.")}}
 
-[Unassigned P-high nightly regressions](https://github.com/rust-lang/rust/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3Aregression-from-stable-to-nightly+label%3AP-high+no%3Aassignee)
+[Unassigned P-high nightly regressions](https://github.com/rust-lang/rust/issues?q=is%3Aopen+label%3Aregression-from-stable-to-nightly+label%3AP-high+no%3Aassignee+-label%3AT-infra+-label%3AT-libs+-label%3AT-relase+-label%3AT-rustdoc+-label%3AT-core)
```
Comment (spastorino) [inline templates/prioritization_agenda.tt:96]: typo, relase -> release

**Review by spastorino — COMMENTED** [review]
:+1: to this change but I'd make it extensive to all the template.

**Comment by apiraino** [comment]
ok I think we're done here. I've compared (more or less thoroughly) all the queries against the compiler agenda template

**Review by spastorino — APPROVED** [review]

**Comment by apiraino** [comment]
replaced by #1175

**Pull request closed without merging**